app/api/scrypt.native.temp.ts

```diff
@@ -0,0 +1,36 @@
+import rnScrypt from 'react-native-scrypt'
```

Comment:
when I name this file as scrypt.native.ts

default platform detected as ios even running on browser for development. Need to figure out how to fix it